### PR TITLE
2 other bugs have been fixed and are ready for review

### PR DESCRIPTION
**Pull request by Ebubeker**
## Description
I managed to fix the icon problem with Elias help that we were facing when adding the tram stops from the right panel. I also added the color changing functionality to the vector point layers settings.

## Motivation and Context
Improves the user experience while accessing our web GOAT software service

## How Has This Been Tested?
Still in review

### CODE REVIEW

File: app/client/src/style/OlStyleDefs.js [hunk 1]
```diff
@@ -951,6 +951,7 @@ export function poisAoisStyle(feature) {
   const icon = poiIconConf.icon;
   if (!poisAoisStyleCache[icon + color]) {
     // Font style
+    console.log(poiIconConf);
```
Comment (EPajares) [inline app/client/src/style/OlStyleDefs.js:954]: Do we need this console log?

**Review by EPajares — COMMENTED** [review]
Thanks Ebubeker for fixing this. I left only one comment.

File: app/client/src/style/OlStyleDefs.js (hunk 1 above)
Comment (Ebubeker) [reply app/client/src/style/OlStyleDefs.js:954]: No I forgot to remove it, sorry

**Review by majkshkurti — APPROVED** [review]

File: app/client/src/components/layers/changeStyle/IconVectorStyle.vue [hunk 2]
```diff
@@ -3,17 +3,34 @@
     <v-tabs grow v-model="tab" style="width: 400px; margin:auto;">
       <v-tab :key="1">
         <v-badge>
-          <b>Change Icon Size</b>
+          <b>Color</b>
```
Comment (majkshkurti) [inline app/client/src/components/layers/changeStyle/IconVectorStyle.vue:6]: Can we add this and other texts to the translation?

**Review by majkshkurti — COMMENTED** [review]
Thanks Ebubeker. Looks great.

File: app/client/src/components/layers/changeStyle/IconVectorStyle.vue (hunk 2 above)
Comment (Ebubeker) [reply app/client/src/components/layers/changeStyle/IconVectorStyle.vue:6]: Yes of course, I will add it now

**Review by EPajares — APPROVED** [review]

**Pull request merged**